{
  "exploitdb": [],
  "itw": [
    {
      "id": "CVE-2023-37788",
      "description": "goproxy v1.1 was discovered to contain an issue which can lead to a Denial of service (DoS) via unspecified vectors.",
      "referenceURL": "https://nvd.nist.gov/vuln/detail/CVE-2023-37788",
      "reportURL": "https://github.com/elazarl/goproxy/issues/502",
      "timeStamp": "2023-07-27T04:05:00Z"
    }
  ],
  "trickest": [
    {
      "id": "CVE-2023-37788",
      "url": "https://github.com/elazarl/goproxy",
      "added_at": "1970-01-01T00:00:00Z"
    },
    {
      "id": "CVE-2023-37788",
      "url": "https://github.com/elazarl/goproxy/issues/502",
      "added_at": "2025-04-09T16:09:20Z"
    }
  ],
  "nomisec": [],
  "nuclei": [],
  "metasploit": [],
  "holloways": [],
  "openpoc": [
    {
      "cve": "CVE-2023-37788",
      "url": "https://github.com/elazarl/goproxy",
      "added_at": "1970-01-01T00:00:00Z",
      "score": 0,
      "template_for": ""
    },
    {
      "cve": "CVE-2023-37788",
      "url": "https://github.com/elazarl/goproxy/issues/502",
      "added_at": "2023-07-27T04:05:00Z",
      "score": 0.4,
      "template_for": ""
    }
  ]
}
